packages/scripts/config/webpack.config.js

```diff
@@ -118,8 +104,11 @@ const config = {
 					test: /[\\/]style(\.module)?\.(sc|sa|c)ss$/,
 					chunks: 'all',
 					enforce: true,
-					name( module, chunks, cacheGroupKey ) {
-						return `${ cacheGroupKey }-${ chunks[ 0 ].name }`;
+					name( _, chunks, cacheGroupKey ) {
+						const chunkName = chunks[ 0 ].name;
```

Comment:
We have to account for subfolders now. We missed that before which forces developers to use a flat structure in the `build` folder.